README.md

```diff
@@ -109,18 +109,18 @@ It's also possible to run the moveit\_ci script locally, without Travis. We demo
 First clone the repo you want to test:
 
     cd /tmp/travis   # any working directory will do
-    git clone https://github.com/ros-planning/moveit2
+    git clone https://github.com/acutronicrobotics/moveit2
     cd moveit2
 
 Next clone the CI script:
 
-    git clone -b ros2 https://github.com/ros-planning/moveit_ci .moveit_ci
+    git clone -b ros2 https://github.com/acutronicrobotics/moveit_ci .moveit_ci
 
 Manually define the variables, Travis would otherwise define for you. These are required:
 
     export TRAVIS_BRANCH=master
     export ROS_DISTRO=crystal
-    export ROS_REPO=ros
+    export ROS_REPO=acutronicrobotics
```

Comment:
Do you have your own debian repo?

Reply:
We're using this variable to download the docker image.

Reply:
Let's get the necessary fixes into the ros-planning/moveit2 dockerfile and revert this change